source/scss/_pf-component-view-selector.scss

```diff
@@ -0,0 +1,23 @@
+//
+// View Selector component
+//
+
+.pf-view-selector {
```

Comment:
Ah, yes - another question. I didn't actually need to style anything there but it seemed like I should class the entire pattern. (especially since I do have "child" classes) So do I keep the class in the html, or not?